docs/networking.md

```diff
@@ -134,6 +134,28 @@ $ kops create cluster \
 ```
 
 The above will deploy a daemonset installation which requires K8s 1.4.x or above.
+##### Enable Kubernetes Data Store for Calico
+By default Calico uses etcd directly for all of it's state. This has the unfortunate side-effect of requiring all nodes to have access to etcd directly which is a security problem. Unfortunately because calico-node runs with `hostNetwork: true`, there's no clean way of using Network Policy to restrict this to specific pods / namespaces. A solution for this is to have calico-node speak directly to the Kubernetes API server and use Third Party Resources to store it's state. 
+
+To enable the Kubernetes Data Store for Calico, you must edit the cluster spec that Kops produces after running `kops create cluster ...`
+
+`kops edit cluster`  will show you a block like this:
+
+```
+  networking:
+    calico: {}
+```
+
+You will need to change that block, and add an additional field, to look like this:
+
+```
+  networking:
+    calico:
+      kubernetesDataStore: true
```

Comment:
I'd suggest switching this to something like:
```
datastore: kubernetes | etcd
```

Just in case someone wants a third option there some day.